admin/kernel-masterdata-service/src/test/java/io/mosip/kernel/masterdata/test/utils/MasterdataSearchHelperTest.java

```diff
@@ -148,7 +170,6 @@ public void setup() {
 		optionalFilterArray = new OptionalFilter[1];
 		optionalFilterArray[0] = new OptionalFilter(Arrays.asList(betweenfilter));
 	}
-
```

Comment:
why you removed that line space between two methods?

Reply:
Added Line spaces between the methods.